README.md

```diff
@@ -40,15 +40,18 @@ For information about the HAL and how to use it in your own projects, please ref
 
 ### High-Power Cores
 
-|     Crate     |                   Documentation                    | Technical Reference Manual |             Target             |
-| :-----------: | :------------------------------------------------: | :------------------------: | :----------------------------: |
-|  [esp32-hal]  |   [![esp32-hal-docs]](https://docs.rs/esp32-hal)   |          [ESP32]           |    `xtensa-esp32-none-elf`     |
-| [esp32c2-hal] | [![esp32c2-hal-docs]](https://docs.rs/esp32c2-hal) |         [ESP32-C2]         | `riscv32imc-unknown-none-elf`  |
-| [esp32c3-hal] | [![esp32c3-hal-docs]](https://docs.rs/esp32c3-hal) |         [ESP32-C3]         | `riscv32imc-unknown-none-elf`  |
-| [esp32c6-hal] | [![esp32c6-hal-docs]](https://docs.rs/esp32c6-hal) |         [ESP32-C6]         | `riscv32imac-unknown-none-elf` |
-| [esp32h2-hal] | [![esp32h2-hal-docs]](https://docs.rs/esp32h2-hal) |         [ESP32-H2]         | `riscv32imac-unknown-none-elf` |
-| [esp32s2-hal] | [![esp32s2-hal-docs]](https://docs.rs/esp32s2-hal) |         [ESP32-S2]         |   `xtensa-esp32s2-none-elf`    |
-| [esp32s3-hal] | [![esp32s3-hal-docs]](https://docs.rs/esp32s3-hal) |         [ESP32-S3]         |   `xtensa-esp32s3-none-elf`    |
+|     Crate     |                   Documentation                    | Technical Reference Manual |             Target             |  Toolchain  |
+| :-----------: | :------------------------------------------------: | :------------------------: | :----------------------------: | :---------: |
+|  [esp32-hal]  |   [![esp32-hal-docs]](https://docs.rs/esp32-hal)   |          [ESP32]           |    `xtensa-esp32-none-elf`     |   ![esp]    |
+| [esp32c2-hal] | [![esp32c2-hal-docs]](https://docs.rs/esp32c2-hal) |         [ESP32-C2]         | `riscv32imc-unknown-none-elf`  |  ![nightly] |
+| [esp32c3-hal] | [![esp32c3-hal-docs]](https://docs.rs/esp32c3-hal) |         [ESP32-C3]         | `riscv32imc-unknown-none-elf`  |  ![nightly] |
+| [esp32c6-hal] | [![esp32c6-hal-docs]](https://docs.rs/esp32c6-hal) |         [ESP32-C6]         | `riscv32imac-unknown-none-elf` |  ![nightly] |
+| [esp32h2-hal] | [![esp32h2-hal-docs]](https://docs.rs/esp32h2-hal) |         [ESP32-H2]         | `riscv32imac-unknown-none-elf` |  ![nightly] |
+| [esp32s2-hal] | [![esp32s2-hal-docs]](https://docs.rs/esp32s2-hal) |         [ESP32-S2]         |   `xtensa-esp32s2-none-elf`    |   ![esp]    |
+| [esp32s3-hal] | [![esp32s3-hal-docs]](https://docs.rs/esp32s3-hal) |         [ESP32-S3]         |   `xtensa-esp32s3-none-elf`    |   ![esp]    |
+
+[esp]: https://img.shields.io/badge/rustc-esp-red.svg
+[nightly]: https://img.shields.io/badge/rustc-nightly%202023/11/30+-red.svg 
```

Comment:
```suggestion
[nightly]: https://img.shields.io/badge/rustc-nightly%202023/11/30-red.svg 
```

Reply:
Yes, the + was intentional to specify the date or later.

Reply:
Then I would maybe suggest updating the header to MSRV and adding also the version for `esp`. Unless we drop the MSRV section as Jesse suggested